CHANGELOG.md

```diff
@@ -40,19 +78,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
-- **CUMULUS-3319**
```

Comment:
These tickets were improperly merged into main CL prior to the release, this was remedied in the V17 release PR.